build/images/scripts/install_cni

```diff
@@ -46,10 +46,20 @@ if [[ ! " ${binaries[*]} " =~ " bandwidth " ]]; then
   install -m 755 /opt/cni/bin/bandwidth /host/opt/cni/bin/bandwidth
 fi
 
+FILE_MODE="0644"
+if [[ -n "${CONFIG_FILE_MODE}" ]]; then
+  if [[ "${CONFIG_FILE_MODE}" =~ ^0?[0-7]{3}$ ]]; then
```

Comment:
it may not even need the check as `install` will validate the permission anyway, this can be simplied as:
```
FILE_MODE="${CONFIG_FILE_MODE:-644}"

install -m "${FILE_MODE}" /etc/antrea/antrea-cni.conflist /host/etc/cni/net.d/10-antrea.conflist
```

Reply:
or even just:
```
install -m "${CONFIG_FILE_MODE:-644}" /etc/antrea/antrea-cni.conflist /host/etc/cni/net.d/10-antrea.conflist
```

Reply:
Updated.